From f808b6dd49dbea3f222e080d835a2f3e182b71f1 Mon Sep 17 00:00:00 2001
From: Meticulus <theonejohnnyd@gmail.com>
Date: Sat, 24 Jun 2017 14:03:01 -0500
Subject: [PATCH] "0001-validus-add-telephony-ext-to-BOOTCLASS_PATH
 _______________________________

*Previous From 3862b3c6542997bff60f94342e005801e93cd965 Mon Sep 17 00:00:00 2001
*Previous From: Meticulus <theonejohnnyd@gmail.com>
*Previous Date: Sat, 24 Jun 2017 13:54:18 -0500
*Previous Subject: [PATCH] validus: add telephony-ext to BOOTCLASS_PATH
*
*-RIL does not work with out telephony-ext"
---
 target/product/core_minimal.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/target/product/core_minimal.mk b/target/product/core_minimal.mk
index 4892047..099e700 100644
--- a/target/product/core_minimal.mk
+++ b/target/product/core_minimal.mk
@@ -76,6 +76,7 @@ PRODUCT_PACKAGES += \
     screencap \
     sensorservice \
     telephony-common \
+    telephony-ext \
     uiautomator \
     uncrypt \
     voip-common \
@@ -96,6 +97,7 @@ PRODUCT_BOOT_JARS := \
     ext \
     framework \
     telephony-common \
+    telephony-ext \
     voip-common \
     ims-common \
     apache-xml \
-- 
1.9.1

